mne/viz/raw.py

```diff
@@ -510,6 +510,7 @@ def _prepare_mne_browse_raw(params, title, bgcolor, color, bad_color, inds,
     """Helper for setting up the mne_browse_raw window."""
     import matplotlib.pyplot as plt
     import matplotlib as mpl
+    plt.ion()
```

Comment:
It was throwing warnings without it. I presumed it would be the right thing to add since this is quite interactive. I think it's fine to leave it here and turn interactive mode off on close. What do you think?